index.html

```diff
@@ -44,23 +44,20 @@
         <dl class="info-headings">
           <div>
             <dt>Data Gathered Since</dt>
-              <!-- @todo get date-start from store -->
-            <dd id="date-start">
-              <time datetime="2017-06-13T11:00">June 13, 2017</time>
+            <dd>
+              <time id="data-gathered-since" datetime="2017-06-13T11:00"></time>
             </dd>
           </div>
           <div>
             <dt>You Have Visited</dt>
-            <!-- @todo get num-sites-visited from store -->
             <dd>
-              <var id="num-sites-visited">6</var> Sites
+              <var id="num-sites-visited">0</var> Sites

```

Comment:
I think the default would be better blank, perhaps not even showing the word "sites" until it's populated. What do you think?